.github/workflows/ci.yml

```diff
@@ -64,6 +64,10 @@ jobs:
       - build_test
       - cypress
     runs-on: ubuntu-latest
+    environment:
+      # The name cannot 
```

Comment:
You are leaving the reader hanging here... :)

Reply:
Good point. I fixed it up so now it is dynamic computed.